apps/api/src/domain/models/context/commands/translate-note/translate-note.command.integration.spec.ts

```diff
@@ -0,0 +1,208 @@
+    payload: {
+        languageCode: originalLanguageCode,
+    },
+});
+
+const noteTranslated = noteAboutReourceCreated.andThen<NoteTranslated>({
+    type: 'NOTE_TRANSLATED',
+    payload: {
+        aggregateCompositeIdentifier: edgeConnectionCompositeIdentifier,
+        languageCode: translationLanguageCode,
+        text: translationNoteText,
+    },
+});
+
+const validPayload: TranslateNote = {
+    aggregateCompositeIdentifier: edgeConnectionCompositeIdentifier,
+    languageCode: translationLanguageCode,
+    text: translationNoteText,
+};
+
+const validCommandFSA = {
+    type: commandType,
+    payload: validPayload,
+};
+
+const fsaFactory = new DummyCommandFsaFactory(() => validCommandFSA);
+
+describe(commandType, () => {
+    let app: INestApplication;
+
+    let databaseProvider: ArangoDatabaseProvider;
+
+    let testRepositoryProvider: TestRepositoryProvider;
+
+    let commandHandlerService: CommandHandlerService;
+
+    let idManager: IIdManager;
+
+    let commandAssertionDependencies: CommandAssertionDependencies;
+
+    beforeAll(async () => {
+        ({ testRepositoryProvider, commandHandlerService, idManager, app, databaseProvider } =
+            await setUpIntegrationTest({
+                ARANGO_DB_NAME: generateDatabaseNameForTestSuite(),
+            }).catch((error) => {
+                throw error;
+            }));
+
+        commandAssertionDependencies = {
+            testRepositoryProvider,
+            idManager,
+            commandHandlerService,
+        };
+
+        await app.get(DynamicDataTypeFinderService).bootstrapDynamicTypes();
+    });
+
+    beforeEach(async () => {
+        await testRepositoryProvider.testSetup();
+    });
+
+    afterAll(async () => {
+        await app.close();
+
+        databaseProvider.close();
+    });
+
+    describe(`when the command is valid`, () => {
+        it(`should succeed with the expected updates to the database`, async () => {
+            await assertCommandSuccess(commandAssertionDependencies, {
+                systemUserId: dummySystemUserId,
+                buildValidCommandFSA: () => validCommandFSA,
+                seedInitialState: async () => {
+                    const eventHistory = noteAboutReourceCreated.as(
+                        edgeConnectionCompositeIdentifier
+                    );
+
+                    await testRepositoryProvider
+                        .getEdgeConnectionRepository()
+                        .create(
+                            EdgeConnection.fromEventHistory(
+                                eventHistory,
+                                edgeConnectionId
+                            ) as EdgeConnection
+                        );
+
+                    await app.get(ArangoEventRepository).appendEvents(eventHistory);
+                },
+                checkStateOnSuccess: async () => {
+                    const edgeConnectionnSearchResult = await testRepositoryProvider
```

Comment:
sp. `edgeConnectionSearchResult` (extra in Connectionn)

Reply:
done